src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ onTabSelected, tabs, selectedTabId }) => {
+  const selectedTab = tabs.find(el => el.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={cn({ 'is-active': tab.id === selectedTab.id })}
+            >
+              <a
+                onClick={() => {
+                  if (tab.id !== selectedTab.id) {
+                    onTabSelected(tab);
+                  }
+                }
+              }
```

Comment:
You don't need to check it twice. You've already checked it in the App component 
```suggestion
                onClick={() => onTabSelected(tab)}
```